### DISCUSSION TITLE
Dividing items

### DISCUSSION THREAD

**Discussion by Anniepoo** [h5gs/wallacepos#5] [Ideas]
I live in the Netherlands, and own a couple shops in Nairobi.
The culture there is kagogoo - 'small stuff' - buying only whats immediately needed.
So, for example, we buy packs of 20 cigars, and then sell them 15KSh for one or 20KSh for 2.
Same with gum, detergent, etc etc
So the problem is, keeping inventory is kind of hard in this, or pricing is hard -  

I'm thinking of making a PR that does what we need for our use case, but am wondering if we can make it more general.

eg. "buy 2, get 1 free"

Probably also should think about things sold in bulk - cloth by 1/8 yard, veggies sold by the Kg,  and stuff like
"if you buy oil we give you a paper funnel, or we sell them for 25 cents"

So I thought I'd stick a note in here to get consensus how this should work before submitting a PR.

**Comment by Anniepoo** [comment]
There's a 'default quantity' - I'm wondering how it should interact with this.
I'm not sure what the use case is for that.  Maybe things like hubcaps, where you'd buy 4, but occasionally somebody would need a replacement?

**Reply by h5gs** [reply]
@Anniepoo are you still interested in this project?

**Comment by mckaygerhard** [comment]
i pretty understand that, such change will need a hard work, if you are still interested in this project @Anniepoo  please go ahead with pull request and i will try to help you.. for now i will focus my effors in the php 7.X working wiout remove older support as firsts steps
